masterlist.yaml

```
@@ -6920,9 +6920,14 @@ plugins:
       - 'WACCF_Armor and Clothing Extension.esp'
       - 'Weapons Armor Clothing & Clutter Fixes.esp'
     msg:
+      - <<: *patchProvided
+        subs: [ 'Complete Alchemy & Cooking Overhaul' ]
+        condition: 'active("Complete Alchemy & Cooking Overhaul.esp") and not active("CLLF - CACO Patch.esp")'
       - <<: *patchProvided
         subs: [ 'Cutting Room Floor' ]
         condition: 'active("Cutting Room Floor.esp") and not active("CLLF - Cutting Room Floor Patch.esp")'
+  - name: 'CLLF - Cutting Room Floor Patch.esp'
+    req: [ 'Containers and Leveled Lists Fixes.esp' ]
```

WACCF and Priest vendor patches are also missing a master, so those should be added as well.

```suggestion
  - name: 'CLLF - Cutting Room Floor Patch.esp'
    req: [ 'Containers and Leveled Lists Fixes.esp' ]
  - name: 'CLLF - WACCF Patch.esp'
    req: [ 'Containers and Leveled Lists Fixes.esp' ]
  - name: 'Priest Vendors - CLLF Integration.esp'
    req: [ 'Containers and Leveled Lists Fixes.esp' ]
```